src/main/java/uk/gov/hmcts/reform/civil/model/dq/Respondent1DQ.java

```diff
@@ -44,8 +44,11 @@ public class Respondent1DQ implements DQ {
     private List<Element<AccountSimple>> respondent1BankAccountList;
     private HomeDetails respondent1DQHomeDetails;
     private YesOrNo respondent1DQCarerAllowanceCredit;
+    private YesOrNo respondent1DQCarerAllowanceCreditFullAdmission;
     private List<Element<RecurringIncomeLRspec>> respondent1DQRecurringIncome;
+    private List<Element<RecurringIncomeLRspec>> respondent1DQRecurringIncomeFA;
     private List<Element<RecurringExpenseLRspec>> respondent1DQRecurringExpenses;
+    private List<Element<RecurringExpenseLRspec>> respondent1DQRecurringExpensesFA;
```

Comment:
Why are these fields added here?

Reply:
The field respondent1DQCarerAllowanceCredit used in part admit journey and respondent1DQCarerAllowanceCreditFullAdmission is used for full admit journey, so I think it will be better to put them together in DQ